assets/dns-app/resources/dns.yaml

```diff
@@ -272,4 +273,12 @@ spec:
     protocol: UDP
   - name: dns-tcp
     port: 53
-    protocol: TCP
\ No newline at end of file
+    protocol: TCP
+---
+apiVersion: scheduling.k8s.io/v1beta1
+kind: PriorityClass
+metadata:
+  name: dns-high-priority
+value: 1000000
```

Comment:
This looks arbitrary tbh. After reading the documentation on priority classes, I stumbled on `system-cluster-critical` and `system-node-critical` built-in priority classes. It would make more sense to use these instead of using arbitrary priority values.

Reply:
I've replaced priority with default `system-cluster-critical`.